File: app/src/main/java/com/metahumanz/pacilread/sync/SyncDiffItem.java
package com.metahumanz.pacilread.sync;

public class SyncDiffItem {
    public static final String STATUS_LOCAL = "local";
    public static final String STATUS_REMOTE = "remote";
    public static final String STATUS_CONFLICT = "conflict";
    public static final String STATUS_UNCHANGED = "unchanged";

    public String entityType;
    public String key;
    public String title;
    public String status;
    public String summary;
    public long localUpdatedAt;
    public long remoteUpdatedAt;
}

File: app/src/main/kotlin/com/metahumanz/pacilread/sync/SyncDiffItem.kt
package com.metahumanz.pacilread.sync

class SyncDiffItem {
    @JvmField var entityType: String? = null
    @JvmField var key: String? = null
    @JvmField var title: String? = null
    @JvmField var status: String? = null
    @JvmField var summary: String? = null
    @JvmField var localUpdatedAt: Long = 0
    @JvmField var remoteUpdatedAt: Long = 0

    companion object {
        const val STATUS_LOCAL = "local"
        const val STATUS_REMOTE = "remote"
        const val STATUS_CONFLICT = "conflict"
        const val STATUS_UNCHANGED = "unchanged"
    }
}
